cmd/consts/dymension.go

```diff
@@ -17,7 +17,8 @@ var BlumbusHubData = HubData{
 	RpcUrl:        "https://rpc-blumbus.mzonder.com:443",
 	ArchiveRpcUrl: "https://rpc-blumbus-archive.mzonder.com:443",
 	GasPrice:      "20000000000",
-	DaNetwork:     CelestiaTestnet,
+	DaNetwork:     AvailTestnet,
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable